api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -117,6 +361,27 @@ message RateLimitPerRoute {
     IGNORE = 2;
   }
 
+  enum RateLimitsOptions {
+    // Use the rate limits options defined in `VhRateLimitsOptions`.
```

Comment:
Let's change this comment such that it will still make sense if/when `VhRateLimitsOptions` eventually goes away.  It can say something like "Client-defined default, typically `OVERRIDE_POLICY`.  If `VhRateLimitsOptions` is set, that will be used instead."